bootstrap/peer.go

```diff
@@ -225,6 +225,14 @@ func (peer *Peer) Close() error {
 		}
 	}
 
+	if peer.BootstrapWeb.Endpoint != nil {
+		errlist.Add(peer.BootstrapWeb.Endpoint.Close())
+	} else {
+		if peer.BootstrapWeb.Endpoint != nil {
+			errlist.Add(peer.Public.Listener.Close())
```

Comment:
Why is this closing the public listener when it's not registered to it?

Reply:
doh, my bad. It seems that we have the same issue in satellite/peer Close method

if peer.Console.Endpoint != nil {
		errlist.Add(peer.Console.Endpoint.Close())
	} else {
		if peer.Console.Endpoint != nil {
			errlist.Add(peer.Public.Listener.Close())
		}
	}

i will remove redundant code from both methods

Reply:
Note, it should close the appropriate listener nevertheless. `peer.Web.Listener`